### PR TITLE
delete the domain in image created for the test

### PR DESCRIPTION
**Pull request by vanajamukkara**
delete the domain in image created for the test at the end of the test

### CODE REVIEW

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java [hunk 1]
```diff
@@ -717,6 +717,15 @@ public void destroy() throws Exception {
     LoggerHelper.getLocal().log(Level.INFO,
         "command to delete domain " + cmd + " \n returned " + output);
     verifyDomainDeleted(replicas);
+
+  }
+
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java]: javadoc missing

Comment (vanajamukkara) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java]: added

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItDomainInImage.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Copyright (c) 2018, 2019, 2020, Oracle Corporation and/or its affiliates.  All rights reserved.
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/kubernetes/operator/ItDomainInImage.java]: This should just be 2018, 2020

File: integration-tests/src/test/java/oracle/kubernetes/operator/ItMultipleClusters.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Copyright (c) 2018, 2019, 2020, Oracle Corporation and/or its affiliates.  All rights reserved.
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/kubernetes/operator/ItMultipleClusters.java]: Here too

**Pull request merged**